nss-day-cohort-47/tabloidfullstack-fiddlesticks
# Lm view comments

# Description
- Applied styling to login, register and nav bar
- All comments appearing when you go to post > post details > view all comments

Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Please go to posts, select a post and then click on the 'view all comments' button. This should open all comments for that post. You can try '/comment/GetByPostId/1' which has multiple comments. 

# Checklist:
